apps/web/src/lang/fr.json

```diff
@@ -8136,20 +8136,182 @@
     "defaultMessage": "Il semble que vous n’ayez pas encore ajouté d’expérience de formation à votre curriculum vitæ .",
     "description": "Alert message informing user to add education experience in application education page."
   },
-  "60tHBx": {
-    "defaultMessage": "Obtention réussie d’un diplôme d’études secondaires ou Programme d’évaluation en education Générale (PEG) equivalent.",
+  "tJLZYs": {
+    "defaultMessage": "Obtention d’un diplôme d’un programme de deux ans ou plus offert par <link>une institution postsecondaire reconnue</link>. Le programme doit compter une spécialisation en sciences informatiques, en technologie de l’information, en gestion de l’information ou dans un autre domaine spécialisé pertinent à la présente publication.",
+    "description": "Descriptive text explaining a valid post secondary education."
+  },
+  "2je6Bi": {
+    "defaultMessage": "Expérience minimale ou formation équivalente",
+    "description": "Heading for education requirements section of the application review page."
+  },
+  "2lOey6": {
+    "defaultMessage": "Votre CV",
+    "description": "Heading for resume section of the application review page."
+  },
+  "4ii2WZ": {
+    "defaultMessage": "Révisez votre candidature et envoyez-la!",
+    "description": "Subtitle for the application review page."
+  },
+  "5A0a7w": {
+    "defaultMessage": "Modifiez la liste des questions de sélection",
+    "description": "Edit link text for screening questions section of the application review page."
+  },
+  "6QGFk9": {
+    "defaultMessage": "Modifiez la liste des compétences requises",
+    "description": "Edit link text for skill requirements section of the application review page."
+  },
+  "7vOa71": {
+    "defaultMessage": "Nous avons bien reçu votre candidature",
+    "description": "Success message after submission for the application review page."
+  },
+  "DXtgrn": {
+    "defaultMessage": "Révisez votre candidature et envoyez-la",
+    "description": "Link text for the application review page."
+  },
+  "A9Urm+": {
+    "defaultMessage": "Lorsque vous envoyez votre candidature, une copie de votre profil est automatiquement créée.",
+    "description": "List item note 1 for the application review page."
+  },
+  "GXmpsW": {
+    "defaultMessage": "Avant de poursuivre, il est important que vous compreniez les remarques suivantes concernant votre candidature :",
+    "description": "Starting message for the application review page."
+  },
+  "JxDtt+": {
+    "defaultMessage": "Révisez votre candidature",
+    "description": "Page title for the application review page."
+  },
+  "MZrE40": {
+    "defaultMessage": "Vous avez dit répondre aux <strong>exigences minimales d’expérience ou de formation (deux années d’études postsecondaires)</strong>  et posséder l’expérience professionnelle suivante :",
```

Comment:
```suggestion
    "defaultMessage": "Vous avez dit répondre aux <strong>exigences minimales d’expérience ou de formation (deux années d’études postsecondaires)</strong> et posséder l’expérience professionnelle suivante :",
```

Reply:
Done in [cleanup french strings](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6668/commits/49e1e777e42041f282522d72d7886ae146162339)